src/NHibernate.Test/NHSpecificTest/ProxyValidator/Fixture.cs

```diff
@@ -183,13 +194,15 @@ public class InvalidInternalField : ValidClass
 		[Test]
 		public void NonVirtualInternal()
 		{
-			Assert.Throws<InvalidProxyTypeException>(() => Validate(typeof(InvalidNonVirtualInternalProperty)));
+			var errors = pv.ValidateType(typeof(InvalidNonVirtualInternalProperty));
+			Assert.That(errors, Has.Count.EqualTo(2));
 		}
 
 		[Test]
 		public void InternalField()
 		{
-			Assert.Throws<InvalidProxyTypeException>(() => Validate(typeof(InvalidInternalField)));
+			var errors = pv.ValidateType(typeof(InvalidInternalField));
+			Assert.That(errors, Has.Count.EqualTo(1));
 		}
 
 		public class InvalidNonVirtualProtectedProperty : ValidClass
```

Comment:
That name is confusing, even more with the withdrawal of the assert message.

```suggestion
		public class ValidNonVirtualProtectedProperty : ValidClass
```

Reply:
Thanks